lib/ui/screens/offer_food_screen.dart

```diff
@@ -149,14 +149,7 @@ class _OfferFoodScreenState extends State<OfferFoodScreen> {
                           }
                         },
                       ),
-                      const SizedBox(height: GapSize.m),
-                      ZOClickableText(
-                        clickableText: context.l10n!.manualName,
-                        prefixText: context.l10n!.consent,
```

Comment:
Could you please also remove keys from localization JSON?

Reply:
Of course, I forgot about that 🤦‍♂️ . Thanks